### PR TITLE
Created new assets fields

### PR DESCRIPTION
**Pull request by samhotep**
## Done

- Added the following fields to the assets model
  - Author
  - Title
  - Type
  - Product
  - Portfolio
  - Vertical
  - Salesforce campaign ID
  - Language
  - Tags
  - Copy doc 

## QA

- Check out this feature branch
- Run the site using the command `dotrun`. The site should start without any issues.
- View the site locally in your web browser at: http://0.0.0.0:8017

## Issue / Card

Fixes [WD-13750](https://warthogs.atlassian.net/browse/WD-13750?atlOrigin=eyJpIjoiYjEzYmVhN2YxOTYzNDc2YjgzNzJmZGEyYWZiOTA0MmUiLCJwIjoiaiJ9)

[WD-13750]: https://warthogs.atlassian.net/browse/WD-13750?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webapp/models.py [hunk 1]
```diff
@@ -26,7 +28,19 @@ class Asset(Base):
     __tablename__ = "asset"
 
     id = Column(Integer, primary_key=True)
-    created = Column(DateTime, nullable=False)
+    author = Column(String, nullable=True)
+    title = Column(String, nullable=True)
+    asset_type = Column(String, nullable=True)
+    product = Column(String, nullable=True)
```
Comment (akbarkz) [inline webapp/models.py:34]: this field is very similar to Tags fields, so it should have a similar data type

File: webapp/models.py [hunk 2]
```diff
@@ -26,7 +28,19 @@ class Asset(Base):
...
+    portfolio = Column(String, nullable=True)
+    copydoc = Column(String, nullable=True)
```
Comment (akbarkz) [inline webapp/models.py:36]: from the design I couldn't find a "Copy Doc" field. Did you mean "Google Drive link"?

File: webapp/models.py [hunk 3]
```diff
@@ -26,7 +28,19 @@ class Asset(Base):
     __tablename__ = "asset"
 
     id = Column(Integer, primary_key=True)
-    created = Column(DateTime, nullable=False)
+    author = Column(String, nullable=True)
+    title = Column(String, nullable=True)
+    asset_type = Column(String, nullable=True)
+    product = Column(String, nullable=True)
+    portfolio = Column(String, nullable=True)
```
Comment (akbarkz) [inline webapp/models.py:35]: in the design there is no "Portfolio" field. Can you please point me at it?

**Comment by samhotep** [comment]
I'd made the updates based on the specs - I've now updated them based on the designs instead

File: webapp/models.py (hunk 1 above)
Comment (akbarkz) [reply webapp/models.py:49]: shouldn't this change from "String" to "relationship"?

File: webapp/alembic/versions/52fbe49dfccb_updated_assets_fields.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        "asset", sa.Column("salesforce_campaign_id", sa.String(), nullable=True)
+    )
+    op.add_column("asset", sa.Column("language", sa.String(), nullable=True))
+    op.add_column("asset", sa.Column("updated", sa.DateTime(), nullable=False))
+    op.add_column("redirect", sa.Column("created", sa.DateTime(), nullable=False))
+    op.add_column("redirect", sa.Column("updated", sa.DateTime(), nullable=False))
+    op.add_column("tag", sa.Column("created", sa.DateTime(), nullable=False))
+    op.add_column("tag", sa.Column("updated", sa.DateTime(), nullable=False))
+    op.add_column("token", sa.Column("created", sa.DateTime(), nullable=False))
+    op.add_column("token", sa.Column("updated", sa.DateTime(), nullable=False))
```
Comment (akbarkz) [inline webapp/alembic/versions/52fbe49dfccb_updated_assets_fields.py:42]: why is there a duplication here?

Comment (samhotep) [reply webapp/alembic/versions/52fbe49dfccb_updated_assets_fields.py]: It's actually separate fields, for `created` and `updated` on the `tag` and `token` models. I added them for better record keeping

File: webapp/models.py (hunk 1 above)
Comment (samhotep) [reply webapp/models.py]: Yes, I've updated it

**Review by akbarkz — APPROVED** [review]

**Pull request merged**